string fpath
---
add_furni_gui/FurniWithName[] furni_list